### PR TITLE
Added option for Dockerfile copy

### PR DESCRIPTION
**Pull request by WodansSon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 # Azure Terraform Module Generator
 `generator-az-terra-module` creates base template to start a new terraform module.
 
-![Alt Text](/doc/scaffolding.640x434.gif)
+![Terraform module template walkthrough](/doc/scaffolding.640x434.gif)
```
Comment (metacpp) [inline README.md:8]: Shall we update the GIF to include the optional Dockerfile step ?

**Review by metacpp — APPROVED** [review]

**Comment by metacpp** [comment]
LGTM.

File: generators/app/index.js [hunk 2]
```diff
@@ -13,11 +13,19 @@ module.exports = class extends Generator {
...
+      default: false,
+      desc: 'Include the Dockerfile'
+    });
   }
 
   initializing() {
     this.props = {
-      name: this.options.name || this.appname
+      name: this.options.name || this.appname,
+      docker: ((this.options.docker.toLowerCase() === 'true') || (this.options.docker.toLowerCase() === 'yes')).toString()
```
Comment (JunyiYi) [inline generators/app/index.js:28]: Is it possible that `this.options.docker` is `null`?

File: README.md (hunk 1 above)
Comment (WodansSon) [reply README.md:8]: Yes, the `.gif` needs to be updated to include the new option. I will open a task to track this.

File: generators/app/index.js (hunk 2 above)
Comment (WodansSon) [reply generators/app/index.js:28]: No because it's default value is `False` .

**Pull request merged**